### PR TITLE
MK-1416 Add "Design" field to harmo study with translation and search criterion

### PR DESCRIPTION
**Pull request by Rima-B**

### CODE REVIEW

File: mica-core/src/main/resources/taxonomies/mica-study.yml [hunk 1]
```diff
@@ -2124,3 +2124,14 @@
       "fr": "Une étude d'harmonization."
   "attributes":
     "static": "true"
+- "name": "harmonizationStudyDesign"
+  "title":
+    "en": "Harmonization Study Design"
+    "fr": "Devis de l'étude d'harmonization"
```
Comment (ymarcon) [inline mica-core/src/main/resources/taxonomies/mica-study.yml]: "Devis" => "Conception" (every where)

File: mica-core/src/main/resources/taxonomies/mica-study.yml [hunk 2]
```diff
@@ -2124,3 +2124,14 @@
...
+  "description":
+    "en": "Harmonization Study Design."
+    "fr": "Devis de l'étude d'harmonization."
```
Comment (ymarcon) [inline mica-core/src/main/resources/taxonomies/mica-study.yml]: "harmonization" => "harmonisation"

File: mica-core/src/main/resources/config/study-form/harmonization-definition.json [hunk 3]
```diff
@@ -1 +1,8 @@
-[]
+[
+  {
+    "key": "harmonizationStudyDesign",
```
Comment (ymarcon) [inline mica-core/src/main/resources/config/study-form/harmonization-definition.json]: Could be simplified to "harmonizationDesign"?

**Comment by Rima-B** [comment]
- Changed has been made, please review.
- The correct french translation of "design" is "devis". Checked with science team and collaborators.

**Comment by ymarcon** [comment]
"[devis](http://www.le-dictionnaire.com/definition.php?mot=devis)" is something else...

**Comment by Rima-B** [comment]
Some epidemiological sources are using "Devis d'étude"
https://admission.umontreal.ca/cours-et-horaires/cours/PTM-6674/
http://www.distance.ulaval.ca/etudes/cours/nut-7023-epidemiologie-nutritionnelle
http://theses.ulaval.ca/archimede/fichiers/21368/21368.html

**Comment by ymarcon** [comment]
This is a fr_CA translation...

**Pull request merged**